homeassistant/components/metoffice/quality_scale.yaml

```diff
@@ -0,0 +1,87 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      No custom actions are defined.
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: |
+      No custom actions are defined.
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup:
+    status: exempt
+    comment: |
+      No explicit event subscriptions.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: todo
+  test-before-configure: done
+  test-before-setup:
+    status: done
+    comment: |
+      Implemented implicitly with data update coordinator
+  unique-config-entry: done
+  # Silver
+  action-exceptions:
+    status: done
+    comment: No custom actions are defined
+  config-entry-unloading: done
+  docs-configuration-parameters: done
+  docs-installation-parameters: done
+  entity-unavailable: done
+  integration-owner: done
+  log-when-unavailable:
+    status: done
+    comment: Handled by DataUpdateCoordinator
+  parallel-updates: done
+  reauthentication-flow: done
+  test-coverage: done
```

Comment:
```py
@pytest.fixture
def no_sensor():
    """Remove sensors."""
    with patch(
        "homeassistant.components.metoffice.sensor.async_setup_entry", return_value=True
    ) as mock_setup_entry:
        yield mock_setup_entry
```
Instead of patching out sensor, transform the PLATFORMS to only load weather